doc/api/webcrypto.md

```diff
@@ -338,6 +338,15 @@ filled with random values, and a reference to `typedArray` is returned.
 
 An error will be thrown if the given `typedArray` is larger than 65,536 bytes.
 
+### `crypto.randomUUID()`
+<!-- YAML
+added: REPLACEME
+-->
+
+* Returns: {string}
+
+Generates a random [RFC 4122][] Version 4 UUID.
```

Comment:
```suggestion
Generates a random [RFC 4122][] Version 4 (variant 1) UUID.
```

Reply:
Ok, this is excessive and can cause more confusion perhaps, as the spec linked doesn't define "variant 1", it justs lists all variants explicitly via how they are ecoded. I revoke this.